.puppet-lint.rc

```diff
@@ -1,3 +1,4 @@
 --fail-on-warnings
 --no-parameter_documentation-check
 --no-parameter_types-check
+--no-140chars-check
```

Comment:
The file is managed by modulesync. And in general we try to keep that check enabled. Is there a reason you want to disable it? You didn't touch any of the puppet classes?

Reply:
no reason. It might have done a mistake.

Reply:
enabled 140chars-check